lib/atscppapi/examples/helloworld/HelloWorldPlugin.cc

```diff
@@ -20,16 +20,23 @@
 #include <iostream>
 #include <atscppapi/GlobalPlugin.h>
 #include <atscppapi/PluginInit.h>
-
+#include <../../../../proxy/api/ts/ts.h>
```

Comment:
There are a lot of ts.h functions missing from the CPPAPI.... providing a wrapper for all of them will take some effort.